### ISSUE TITLE
Use strongly-typed debug & code IDs

### ISSUE THREAD

**Issue by gabrielesvelto** [msirringhaus/minidump_writer_linux#11]
As per title, these are being introduced in https://github.com/luser/rust-minidump/pull/432

**Comment by Gankra** [comment]
@Jake-Shadle is this done?

**Comment by Gankra** [comment]
(or coherent?)

**Comment by Jake-Shadle** [comment]
Nope, I think it would help a bit on Linux, but Windows doesn't need it, and Mac is already just going directly to [GUID](https://github.com/rust-minidump/rust-minidump/pull/534/files#diff-8896d41e2ef879015f9341e2add2bf3fffe128fd01f89943fff39d7320378a84R549-R586) since that's the type in the CV record so not super useful there?
